### PR TITLE
Edit/add new pub

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the goal of this pull request and the changes that were made.

Add new publication and update things for end of year

### CODE REVIEW

File: data/projects.yml [hunk 1]
```diff
@@ -923,4 +923,18 @@ project_list:
...
     github_user:
       - lizneely
-    note: "This publication accompanies the exhibition Artful Living: O’Keeffe & Modern Design on view at the O’Keeffe Welcome Center in Abiquiú, New Mexico from April 2, 2024 through January 1, 2026. The publication hs been customized with an extension that utilizes the museum's Linked Art-based linked data infrastructure. This increases discoverability by providing references between the publication and collection information. The publication can be downloaded as a PDF and is available in both English and Spanish."
\ No newline at end of file
+    note: "This publication accompanies the exhibition Artful Living: O’Keeffe & Modern Design on view at the O’Keeffe Welcome Center in Abiquiú, New Mexico from April 2, 2024 through January 1, 2026. The publication hs been customized with an extension that utilizes the museum's Linked Art-based linked data infrastructure. This increases discoverability by providing references between the publication and collection information. The publication can be downloaded as a PDF and is available in both English and Spanish."
+
+  - id: "newfields_pissarro"
+    src: /screenshots/_newfields_pissarro
```
Comment (geealbers) [inline data/projects.yml:929]: file extension in `src`

**Comment by Erin-Cecele** [comment]
Fixed! Checked the preview this time as well. 

Also added two new things.

File: content/about/who-we-are.md [hunk 2]
```diff
@@ -31,70 +31,87 @@ Thank you to all our growing list of contributors who have made Quire what it is
...
-- [@anniemcewen](https://github.com/anniemcewen/)
+- [@annaficek](https://github.com/annaficek)
+- [@anniemcewen](https://github.com/anniemcewen)
 - [@antoinentl](https://github.com/antoinentl)
 - [@aprigge](https://github.com/aprigge)
 - [@audreywarne](https://github.com/audreywarne)
 - [@badragan](https://github.com/badragan)
+- [@basiakapolka](https://github.com/basiakapolka)
 - [@belmendo](https://github.com/belmendo)
+- [bjhewitt](https://github.com/bjhewitt)
```
Comment (geealbers) [inline content/about/who-we-are.md:45]: sorry, just saw this one. missing `@`

Comment (Erin-Cecele) [reply content/about/who-we-are.md]: Eagle eye!! Thank you

**Review by geealbers — APPROVED** [review]

**Pull request merged**